src/newriff.js

```diff
@@ -0,0 +1,85 @@
+const maxApi = require('max-api');
+const scribble = require('scribbletune');
+const _ = require('lodash');
```

Comment:
Can we just use a polyfill for `repeat` coz requiring the entire library like this will inject it in the resulting JS file bloating the file size.

Reply:
Sure, I just wanted to show its possible to do it this way :P